src/containers.zig

```diff
@@ -56,13 +61,15 @@ pub fn ColumnLayout(peer: Callbacks, widgets: []Widget) void {
         }
     }
 
+
+
     var childY: f32 = 0.0;
     for (widgets) |widget, i| {
         const isLastWidget = i == widgets.len - 1;
         if (widget.peer) |widgetPeer| {
             const available = Size{
                 .width = @intCast(u32, peer.getSize(peer.userdata).width),
-                .height = if (widget.container_expanded) childHeight else (@intCast(u32, peer.getSize(peer.userdata).height) - @floatToInt(u32, childY)),
+                .height = if (widget.container_expanded) childHeight else subtract(@intCast(u32, peer.getSize(peer.userdata).height), @floatToInt(u32, childY)),
```

Comment:
```suggestion
(@intCast(u32, peer.getSize(peer.userdata).height) -| @floatToInt(u32, childY))
```
In Zig, `-|` is saturating subtraction which is much shorter (and often faster) than your `substract()` function